app/components/decision/decision-result-select/select-item.hbs

```diff
@@ -1,4 +1,6 @@
-<div class="vlc-color-badge-holder">
-  <div class="vlc-color-badge-holder__badge vlc-color-badge {{this.modifierClass}}"></div>
-  <div class="vlc-color-badge-holder__label">{{@decisionResultCode.label}}</div>
+<div class="auk-u-flex auk-u-flex--align-baseline">
+  <div class="auk-u-flex__item auk-u-flex__item--shrink">
+    <WebComponents::AuColorBadge @skin="{{this.skin}}" />
```

Comment:
Gezien `this.skin` hier sowieso als een string is, zijn de extra quotes hier niet nodig. Dit mag meteen `@skin={{this.skin}}` worden. Ik pas het aan.

Reply:
Goeie opmerking, bedankt!